test/framework/command.go

```diff
@@ -14,33 +14,77 @@
 
 package framework
 
-import "os/exec"
+import (
+	"os/exec"
 
-// ExecuteCommand Execute command in current directory
-func ExecuteCommand(commandName string, args ...string) (string, error) {
-	return ExecuteCommandInDirectory("", commandName, args...)
+	"go.uber.org/zap"
+)
+
+// CreateCommand methods initializes the basic data to run commands.
+func CreateCommand(commandName string, args ...string) Command {
+	return &commandStruct{name: commandName, args: args}
+}
+
+// Command wraps information about the command to execute.
+type Command interface {
+	WithLoggerContext(loggerContext string) Command
+	InDirectory(directory string) Command
+	Execute() (string, error)
+}
+
+// struct that represents a command.
+type commandStruct struct {
+	name          string
+	args          []string
+	directory     string
+	loggerContext string
+}
+
+// WithLoggerContext method attaches a logger context to trace all the command logs when executing it.
```

Comment:
The comments of interface's methods should be in the interface directly

Reply:
Done